lib/rgeo/feature/geometry.rb

```diff
@@ -711,6 +711,18 @@ def +(rhs)
       def *(rhs)
         intersection(rhs)
       end
+
+      # Convenience method to transform/project a geometry
+      # to a different coordinate system from the geometry itself
+      # instead of the cast method.
+      #
+      # @note: Not an OGC SFS method
+      #
+      # @param [RGeo::Feature::Factory] other_factory
+      # @return [RGeo::Feature::Geometry]
+      def transform(other_factory)
+        Feature.cast(self, factory: other_factory, project: true)
```

Comment:
Why do we need to set `project: true` here ?

Reply:
If we don't set `project: true`, it will not try to transform the coordinates and just re-wrap the object from its factory to the given factory.